visual/hexary.go

```diff
@@ -84,3 +84,44 @@ func Vertical(w io.Writer, hex []byte, highlighted int, name string, indexColors
 	>];
 	`)
 }
+
+func Horizzontal(w io.Writer, hex []byte, highlighted int, name string, indexColors []string, fontColors []string, compression int) {
+	fmt.Fprintf(w,
+		`
+	%s [label=<
+	<table border="0" color="#000000" cellborder="1" cellspacing="0">
+	<tr>`, name)
+	if hex[len(hex)-1] == 16 {
+		hex = hex[:len(hex)-1]
+	} else {
+		compression = 0 // No compression for non-terminal keys
+	}
+	for i, h := range hex {
+		if i < len(hex)-compression-2 || i > len(hex)-3 {
+			if i < highlighted {
+				fmt.Fprintf(w,
+					`		<td bgcolor="%s"><font color="%s">%s</font></td>
+		`, indexColors[h], fontColors[h], hexIndices[h])
+			} else {
+				if int(h) > len(indexColors) - 1 {
```

Comment:
I would say remove this fallback, it is better we fail than turn wrong values into `1`s